src/test/java/com/marklogic/spark/reader/file/MlcpArchiveFileReaderTest.java

```diff
@@ -0,0 +1,44 @@
+package com.marklogic.spark.reader.file;
+
+import com.marklogic.spark.AbstractIntegrationTest;
+import com.marklogic.spark.Options;
+import org.apache.spark.sql.DataFrameReader;
+import org.apache.spark.sql.Dataset;
+import org.apache.spark.sql.Row;
+import org.junit.jupiter.api.Test;
+
+import java.util.HashSet;
+import java.util.List;
+
+public class MlcpArchiveFileReaderTest extends AbstractIntegrationTest {
```

Comment:
For consistency with the other tests, name this `ReadMlcpArchiveFilesTest`. 

Also, Sonar is oddly picky about test classes being `public` so remove that keyword as well. You can run `./gradlew sonar` to see the Sonar report, as long as you have Sonar running via Docker.